src/SourceLink.Bitbucket.Git/GetSourceLinkUrl.cs

```diff
@@ -16,7 +16,88 @@ public sealed class GetSourceLinkUrl : GetSourceLinkUrlGitTask
         protected override string HostsItemGroupName => "SourceLinkBitbucketGitHost";
         protected override string ProviderDisplayName => "Bitbucket.Git";
 
-        protected override string BuildSourceLinkUrl(Uri contentUri, Uri gitUri, string relativeUrl, string revisionId, ITaskItem hostItem)
-            => UriUtilities.Combine(UriUtilities.Combine(contentUri.ToString(), relativeUrl), "raw/" + revisionId + "/*");
+        private const string IsEnterpriseEditionMetadataName = "EnterpriseEdition";
+        private const string VersionMetadataName = "Version";
+        private const string VersionWithNewUrlFormat = "4.7";
+
+        protected override bool SupportsImplicitHost => false;
+
+        protected override string BuildSourceLinkUrl(Uri contentUri, Uri gitUri, string relativeUrl, string revisionId,
+            ITaskItem hostItem)
```

Comment:
>ITaskItem hostItem [](start = 12, length = 18)

Style: keep the parameter on the previous line. We either indent all parameters or none.

Reply:
Done